server/pyproject.toml

```diff
@@ -63,6 +63,7 @@ httpx-sse = ">=0.3.1,<0.5.0"
 maturin = "1.7.4"
 patchelf = { version = "^0.17.2.0", markers = "platform_system=='Linux'" }
 poetry-lock-package = ">=0.4.4,<0.6.0"
+poutils = "^0.15.4"
```

Comment:
I would suggest to put that dependency in a dedicated group

Reply:
How would you name this group? `precommit`? `ci`? Would you also move other dependencies such as `deptry` or `sqlfluff` that are used in pre-commit?

After some thought, I am not sure if it is a good idea. If we create another group (say `precommit` or `ci`):
- If you are a developer: you would need to install both `dev` group AND this new group to be able to commit
- If you are the ci: you would also need to install both groups.

So, what would be the benefit of having a different group?

Reply:
> How would you name this group?

> If you are a developer: you would need to install both dev group AND this new group to be able to commit

That's OK for me since that would mean to dev is trying to work on the documentation, which we could add a dedicated step in the documentation for the setup.

> If you are the ci: you would also need to install both groups.

Set once and forget